week8/dongyun/src/components/cart/CartCard.tsx

```diff
@@ -0,0 +1,54 @@
+import * as S from './CartCard.style.tsx';
+import type {cartItem} from "../../types/types.ts";
+
+type CartCardProps = {
+    cartItem: cartItem
+}
+
+const CartCard = ({cartItem}:CartCardProps) => {
+    const editItem = () =>{
+        fetch(`http://localhost:3001/cart-list/${cartItem.id}`, {
+            method:"PATCH",
+            headers: {'Content-Type': 'application/json'},
+            body: JSON.stringify({
+                count: 99
+            })
+        }).then(res => {
+            console.log(res.json())
+        }).catch(error => {
+            console.log(error);
+        });
+    }
```

Comment:
이렇게 표현하는 것도 좋을 것 같습니다!

```suggestion
    const editItem = async () => {
  try {
    const res = await fetch(`http://localhost:3001/cart-list/${cartItem.id}`, {
      method: "PATCH",
      headers: { 'Content-Type': 'application/json' },
      body: JSON.stringify({ count: 99 })
    });
    const data = await res.json();
    console.log(data);
  } catch (error) {
    console.error(error);
  }
};
```

Reply:
넵 개선해보겠습니다. 감사합니다.